features/user_can_delete_post.feature

```diff
@@ -0,0 +1,28 @@
+Feature:
+"As a user,
+In order to get rid of a wrong post,
+I would like to able to delete it."
+
+  Background:
+    Given the following user exists
+    | name     | email             | password     |
+    | Fat Bob  | fatbob@gmail.com  | ilovecatfood |
+    | Slim Jim | slimjum@gmail.com | password123  |
```

Comment:
I think you can simplify this even more. You don't need the email and password in the test. You can allow FactoryBot to set that.

Reply:
Just keep the name

Reply:
Done!